htdocs/comm/propal/class/propal.class.php

```diff
@@ -649,6 +667,27 @@ function updateline($rowid, $pu, $qty, $remise_percent, $txtva, $txlocaltax1=0.0
 		if ($this->statut == self::STATUS_DRAFT)
 		{
 			$this->db->begin();
+   
+			//Fetch current line from the database and then clone the object and set it in $oldline property
+			$line = new PropaleLigne($this->db);
+			$line->fetch($rowid);
+			$line->fetch_optionals(); // Fetch extrafields for oldcopy
+
+			$staticline = clone $line;
+
+			$line->oldline = $staticline;
+			$this->line = $line;
+			$this->line->context = $this->context;
+
+			//Fetch product
+			if (!empty($this->line->fk_product))
```

Comment:
What is goal of fetching product here ?
With previous code, we did not do that, and it seems not used with new code too.

Reply:
Commit was common with the resources branch which required product fetching in both add/update, I will remove if necessary